README.md

```diff
@@ -104,13 +104,30 @@ If a Collector handles both trace and metric data then both must be accounted
 for when sizing. For example, 5K spans per second plus 10K data points per
 second would require 1 CPU core.
 
-The recommendation is to use a ratio of 1:2 for CPU:memory and to allocate at
-least a CPU core per Collector. Multiple Collectors can deployed behind a
-simple round-robin load balancer. Each Collector runs independently, so scale
-increases linearly with the number of Collectors you deploy.
+The recommendation is to use a ratio of 1:2 for CPU:memory. By default, the
```

Comment:
I realized I've misunderstood this ratio and am unsure of the units.  Is this referring to bus speeds?

Reply:
1 CPU to 2GB of memory. I can make this explicit.